descartes_light/solvers/include/descartes_light/solvers/bgl/bgl_solver.h

```diff
@@ -82,6 +82,9 @@ class BGLSolverBaseSVDE : public BGLSolverBase<FloatType>
   BuildStatus buildImpl(const std::vector<typename WaypointSampler<FloatType>::ConstPtr>& trajectory,
                         const std::vector<typename EdgeEvaluator<FloatType>::ConstPtr>& edge_eval,
                         const std::vector<typename StateEvaluator<FloatType>::ConstPtr>& state_eval) override;
+
+protected:
+  std::vector<typename EdgeEvaluator<FloatType>::ConstPtr> edge_eval_{ nullptr };
```

Comment:
```suggestion
  std::vector<typename EdgeEvaluator<FloatType>::ConstPtr> edge_eval_;
```

Probably better to just leave the vector empty